docusaurus.config.js

```diff
@@ -145,13 +168,20 @@ module.exports = {
         theme: {
           customCss: require.resolve('./src/css/custom.css')
         },
-        gtag: {
-          // You can also use your "G-" Measurement ID here.
-          trackingID: 'UA-138955219-1',
-          // Optional fields.
-          anonymizeIP: true // Should IPs be anonymized?
-        }
-      })
+        // lastVersion: 'current',
+        // versions: {
+        //   // Example configuration:
+        //   // <WILL> may have to be fixed
+        //   current: {
+        //     label: 'docs',
+        //     path: 'docs'
+        //   },
+        //   '0.13.9': {
+        //     label: '0.13.9-docs',
+        //     path: '0.13.9'
+        //   }
+        // }
+      }
```

Comment:
```suggestion
      })
```

I think this is where the error is coming from

Reply:
hmm that doesn't seem to fix it (my editor says that the file is valid JS)